applications_tests/gls_sharp_navier_stokes_2d/moving_rectangle.prm

```diff
@@ -0,0 +1,124 @@
+# Listing of Parameters
+# ---------------------
+# --------------------------------------------------
+# Simulation and IO Control
+#---------------------------------------------------
+subsection simulation control
+  set method                  = bdf1
+  set time step               = 0.03125
+  set number mesh adapt       = 3
+  set time end                = 0.125
+  set subdivision             = 1
+end
+#---------------------------------------------------
+# Physical Properties
+#---------------------------------------------------
+subsection physical properties
+  subsection fluid 0
+    set kinematic viscosity            = 1
+  end
+end
+
+#---------------------------------------------------
+# Mesh
+#---------------------------------------------------
+subsection mesh
+    set type                 = dealii
+    set grid type            = subdivided_hyper_rectangle
+    set grid arguments       = 1,1:-2,-2 : 2,2 : true
+    set initial refinement   = 4
+end
+#---------------------------------------------------
+# FEM
+#---------------------------------------------------
+subsection FEM
+    set velocity order        = 1
+    set pressure order        = 1
+end
+
+# --------------------------------------------------
+# Boundary Conditions
+#---------------------------------------------------
+subsection boundary conditions
+    set number                  = 0
+end
+
+# --------------------------------------------------
+# IB particles
+#---------------------------------------------------
+subsection particles
+	set number of particles = 1
+	set stencil order = 6
+	set initial refinement = 2
+	set refine mesh inside radius factor = 0.7
+	set refine mesh outside radius factor = 1.5
+	set integrate motion = false
+	set alpha=0.5
+	set assemble Navier-Stokes inside particles=true
+	subsection particle info 0
+		set type = rectangle
+		set shape arguments = 0.25; 0.5
+		subsection position
+		    set Function expression =sin(t);0.01
+		end
+		subsection velocity
+            set Function expression =0;cos(t)
+        end
+	end
+end
+
+# --------------------------------------------------
+# Mesh Adaptation Control
+#---------------------------------------------------
+subsection mesh adaptation
+  set fraction coarsening  = 0
+  set fraction refinement  = 0
```

Comment:
I set them that way because I want no refinement between time steps, but I can't deactivate it completely since I need initial near-particle refinement.

Reply:
oh ok, interesting, I did not know this mechanism!